src/web_app/js/peerconnectionclient.js

```diff
@@ -213,6 +213,7 @@ PeerConnectionClient.prototype.setRemoteSdp_ = function(message) {
   message.sdp = maybeSetAudioSendBitRate(message.sdp, this.params_);
   message.sdp = maybeSetVideoSendBitRate(message.sdp, this.params_);
   message.sdp = maybeSetVideoSendInitialBitRate(message.sdp, this.params_);
+  message.sdp = maybeRemoveVideoFec(message.sdp, this.params_);
```

Comment:
add `maybeRemoveVideoFec` to https://github.com/webrtc/apprtc/pull/302/files#diff-b6ceaef7892f5905083d3a5ed9e0a10eR11.